rhel/7.5/aws/DCOS-1.10.8/docker-1.13.1.git8633870/dcos_images.yaml

```diff
@@ -0,0 +1,15 @@
+ap-northeast-1: ami-0295f450b9ee52e6a
```

Comment:
This is because the docker that gets installed from the rhel-extras repo is (https://github.com/projectatomic/docker/commit/8633870faeff2c03dd6a75fdef9c2976379339c6) which is not the latest commit in that github repository (commit was merged in September 6th). I wanted a way to differentiate that, so I added the SHA there. I am open to other suggestions though.

Reply:
This is the version that is installed: `docker.x86_64 2:1.13.1-75.git8633870.el7_5`

Reply:
I just assumed that in the future when RH updates the docker version that is available from the rhel-extra repo, we need to be able to differentiate from that.